R-package/README.md

```diff
@@ -354,6 +354,10 @@ rhub::check(
 )
 ```
 
+Alternatively, GitHub Actions can run code above for you. On a pull request, create a comment with this phrase:
+
+> /gha run r-solais
+
```

Comment:
```suggestion
Alternatively, GitHub Actions can run code above for you. On a pull request, create a comment with this phrase:

> /gha run r-solais

**NOTE:** Please only do this only once you see that other R tests on a pull request are passing. R Hub is a free resource with limited capacity, and we want to be respectful community members.
```

Could you please add a note encouraging some restraint in the use of this job?

Reply:
Sure, will add this note.

Reply:
thank you